src/js/claims-status/containers/DocumentRequestPage.jsx

```diff
@@ -97,7 +97,7 @@ class DocumentRequestPage extends React.Component {
                     <Element name="uploadError"/>
                     <Notification title={message.title} body={message.body} type={message.type}/>
                   </div>}
-                <h1 className="claims-header">{trackedItem.displayName}</h1>
+                <h1 className="claims-header">{trackedItem.displayName || "Document Request"}</h1>
```

Comment:
I don't think this will fix the error, since it's trackedItem that is `undefined`, based on the error message.

Reply:
Thanks Jeff, this has been replaced by the following:
```
<h1 className="claims-header">{trackedItem ? trackedItem.displayName : 'Document Request'}</h1>
```
As well as making all other references to trackedItem similar.
In the event that trackedItem is undefined, how should AddFilesForm and its onSubmit prop handle this?

Reply:
This page in general is not very useful if the tracked item doesn't exist. I wonder if we should just redirect back to the claim detail page when that happens?

Reply:
Thanks @jbalboni, good idea, changes made (and unmade).